### PR TITLE
Adding changes to make sure we are compliant with bandit

### PR DESCRIPTION
**Pull request by mfixstsci**
These are the results from the most recent bandit run with the changes I am submitting in this PR. 
Under test results, you can see "No issues identified"
```
bandit -r exoctk/exoctk/ -lll

[main]	INFO	profile include tests: None
[main]	INFO	profile exclude tests: None
[main]	INFO	cli include tests: None
[main]	INFO	cli exclude tests: None
[main]	INFO	running on Python 3.6.7
Run started:2019-07-26 19:04:02.306356

Test results:
	No issues identified.

Code scanned:
	Total lines of code: 7855
	Total lines skipped (#nosec): 0

Run metrics:
	Total issues (by severity):
		Undefined: 0.0
		Low: 20.0
		Medium: 11.0
		High: 0.0
	Total issues (by confidence):
		Undefined: 0.0
		Low: 1.0
		Medium: 4.0
		High: 26.0
Files skipped (0):
```

The changes were very minor but one of them included removing the `debug` feature from the app. If you want to read more about why I did this, look here: https://bandit.readthedocs.io/en/latest/plugins/b201_flask_debug_true.html

Also, here is what I followed to replace the `os.system` commands: https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module
This is a mapping of functions from the `os` library to `pathlib.Path`. 

I didn't test this directly in the app (couldn't think of a clever way) but assuming we are just creating a file or removing a files I did the following test:

```
$ ipython

In [1]: from pathlib import Path 

In [2]: p = Path('/Users/mfix/Desktop/test.txt')                                                                                                                                  

In [3]: p.touch()                                                                                                                                                                 

In [4]: ls /Users/mfix/Desktop/test.txt                                                                                                                                           
/Users/mfix/Desktop/test.txt

In [5]: p.unlink()

In [6]: ls /Users/mfix/Desktop/test.txt                                                                                                                                           
ls: /Users/mfix/Desktop/test.txt: No such file or directory 
```

### CODE REVIEW

**Comment by bourque** [comment]
Thanks to @mfixstsci for the changes!  I think this is ready for review @julesfowler

File: exoctk/exoctk_app/app_exoctk.py [hunk 1]
```diff
@@ -984,4 +984,4 @@ def atmospheric_retrievals():
 if __name__ == '__main__':
     # os.chmod('/internal/data1/app_data/.astropy/cache/', 777)
     port = int(os.environ.get('PORT', 5000))
-    app_exoctk.run(host='0.0.0.0', port=port, debug=True)
+    app_exoctk.run(host='0.0.0.0', port=port)
```
Comment (julesfowler) [inline exoctk/exoctk_app/app_exoctk.py]: I feel great about getting rid of `debug` mode -- `bandit` or otherwise.

File: exoctk/exoctk_app/log_exoctk.py [hunk 2]
```diff
@@ -34,12 +34,15 @@ def create_db(dbpath, overwrite=True):
...
+        p = Path(dbpath)
+        
         # Remove existing file if overwriting
         if os.path.isfile(dbpath) and overwrite:
-            os.system('rm {}'.format(dbpath))
+            p.unlink()
 
         # Make the new file
-        os.system('touch {}'.format(dbpath))
+        p.touch()
```
Comment (julesfowler) [inline exoctk/exoctk_app/log_exoctk.py]: @bourque @mfixstsci 
I like this way of file I/O -- definitely better than `os.system`. I believe when I had looked into this in the past, I had seen that the "pythonic" recommendation was `shutil`. Has `Path` surpassed this and/or is `shutil` lacking functionality?

Comment (mfixstsci) [reply exoctk/exoctk_app/log_exoctk.py]: I don't know if it has surpassed it but I know that this a more secure way of handling the creation and deletion of paths, files etc. I am not sure if `shutil` is or isn't compliant with `bandit`. I believe that `Path` was suggested by Jeff when we met with the exomast folks.

Comment (julesfowler) [reply exoctk/exoctk_app/log_exoctk.py]: @mfixstsci Did some reading -- looks like `pathlib` is being called the summation of `os.path`, `shutil`, and `glob`, so I'm comfy moving forward.
